package.json

```diff
@@ -45,7 +45,7 @@
     "react-dom": ">15.0.0"
   },
   "dependencies": {
-    "prop-types": "^15.5.10",
+    "prop-types": "^15.0.0",
```

Comment:
Technically you would want to do `>15.0.0` like it says in the `peerDependencies` if you are trying to get the matching version.  But I am unsure, if this matters, that just moving this into a peer dep is not the better move.  What is this change trying to solve?

Reply:
One of our services depends on `prop-types@<15.5.0`, and then `npm` complains:

```
npm ERR! peer dep missing: prop-types@>15.5.10, required by @streammedev/flyout@1.0.0
```

I'm just trying to fix the peerDepends error.

Reply:
But now I'm very confused, and not sure this is a good change at all.

Reply:
Hm, I think that is a weird error.  But IMO this should probably change to `>15.0.0`, then you could asses if you want to do the work to get them all aligned on version.